src/kinetics/PlogRate.cpp

```diff
@@ -120,6 +116,14 @@ void PlogRate::setRates(const std::multimap<double, ArrheniusRate>& rates)
         j++;
         rates_.push_back(rate.second);
     }
+    if (!m_valid) {
+        // ensure that reaction rate can be evaluated (but returns NaN)
+        rates_.reserve(2);
+        pressures_[std::log(Tiny)] = {0, 1};
+        rates_.push_back(ArrheniusRate());
+        pressures_[std::log(1. / Tiny)] = {1, 2};
+        rates_.push_back(ArrheniusRate());
```

Comment:
You can just provide a rate constant for a single, arbitrary pressure value -- the two lines below this already handle extrapolating outside the specified pressure range.